### DISCUSSION TITLE
请问如何添加新的3d实体模型-----(How do I add a new 3D solid model?)

### DISCUSSION THREAD

**Discussion by JasperH00** [zouyaoji/vue-cesium#283] [Q&A]
官方文档入门里的地球模型可以导进来,请教一下如何导入其它的模型

The Earth model in the official document entry can be imported, please ask how to import other models

**Comment by zouyaoji** [comment]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/graphics/vc-graphics-model
